renovate.json

```diff
@@ -16,8 +16,13 @@
     {
       "groupName": "all non-major dependencies",
       "groupSlug": "all-minor-patch",
-      "excludePackageNames": ["puppeteer", "ajv", "ansi-regex"],
+      "excludePackageNames": ["puppeteer", "ansi-regex"],
       "matchUpdateTypes": ["minor", "patch"]
+    },
+    {
+      "matchPackageNames": ["ajv"],
+      "matchUpdateTypes": ["major"],
+      "enabled": false
     }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about relying on closed PR rather than adding extra conf?